source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/readonly/MediaRenderer.java

```diff
@@ -230,20 +354,20 @@ private FullscreenVideoView renderMediaPlayer(
                 if (onlineMediaLoader != null)
                 {
                     mediaView.setDataSource(onlineMediaLoader.loadOnlineMedia(mediaSources, new IMediaDataSourceOnPreparedListener() {
-                            @Override
-                            public void prepareMediaPlayer() {
-                                mediaView.prepare();
-                            }
-                        }), isAudio);
+                        @Override
+                        public void prepareMediaPlayer() {
+                            mediaView.prepare();
+                        }
+                    }), isAudio);
                     registeredOnlineMediaLoader = true;
                 }
             }
 
-            if(!registeredOnlineMediaLoader)
+            if (!registeredOnlineMediaLoader)
             {
                 for (int i = 0; i < sourcesSize; i++)
                 {
-                    MediaSource mediaSource = mediaSources.get(i);
+                    io.adaptivecards.objectmodel.MediaSource mediaSource = mediaSources.get(i);
```

Comment:
why do you removed the import io.adaptivecards.objectmodel.MediaSource  and then use full namespace here